src/test/run-make-fulldeps/coverage/lib/used_inline_crate.rs

```diff
@@ -28,12 +28,6 @@ pub fn used_inline_function() {
     }
     use_this_lib_crate();
 }
```

Comment:
Any theories why this is no longer generating the `Unexecuted instantiation`? It probably doesn't have to be documented here anymore (the absence of the unexpected should be expected, so this is good), but I'm curious why it changed.

Here's a trick for keeping the diffs clean in the `expected` results, above. The line numbers in `llvm-cov show` mess things up, so I avoid changing line numbers whenever possible: Just replace these 6 deleted lines with 6 blank lines.

Reply:
The `Unexecuted instantiation` re-appears if I undo the changes in `compiler/rustc_typeck/src/collect.rs`. I'm not exactly sure *how* that causes it to re-appear though.

Reply:
Yeah, I assumed it's related somehow.

Thanks for inserting the blank lines. Makes the expected result diffs a lot easier to read!

Reply:
Do you want me to remove the blank lines before merging?

Reply:
Not necessary. Thanks.